### DISCUSSION TITLE
illegal multibyte sequence

### DISCUSSION THREAD

**Discussion by lautnerxing** [cantools/cantools#424] [Q&A]
![image](https://user-images.githubusercontent.com/51226709/168572147-fab605b0-813c-4109-8377-26f58297b116.png)

**Comment by juleq** [comment]
You could:
- Diff your working and your non working dbc to check what exactly causes the issue
- Open your non-working version in CANdb++. If it can open it, save it again to ensure it is a legal dbc. Diff it to the original non-working version to see if there are differences that CANdb++ fixed for you. If it does not open, fix the dbc using the diff to a working version

**Reply by lautnerxing** [reply]
* There is no difference after comparison
* I changed the file code to UTF-8, but it still doesn't work

**Comment by andlaus** [comment]
can you tell us where the "å" in the file stems from? (e.g., Norwegian or Danish comments)

**Reply by andlaus** [reply]
also, what happens if you load the database with `encoding='latin1'`? (or `cantools generate_c_source -e "latin1" $DBC_FILE` in your case.)
